indico/modules/rb/client/js/modules/bookRoom/filters/RecurrenceForm.jsx

```diff
@@ -105,7 +105,7 @@ export default class RecurrenceForm extends FilterFormComponent {
               onChange={this.onNumberChange}
             />
           </Form.Field>
-          <Form.Field>
+          <Form.Field style={{paddingRight: '0em'}}>
```

Comment:
I think just `0` (as a number) is enough if you want zero, isn't it? (same in the previous change above)